hal/imx_rt.c

```diff
@@ -218,6 +218,16 @@ bootloader_api_entry_t *g_bootloaderTree;
 #define LUT_SEQ_IDX_14                    (0x38U)   /* RESERVED */
 #define LUT_SEQ_IDX_15                    (0x3CU)   /* Dummy */
 
+#ifdef TARGET_library
```

Comment:
We should probably add a quick blurb about this new macro to the docs

Reply:
Sure can do. It's not a new macro, really. When you use the config/examples/library.config the TARGET=library, which does a CFLAG for `-DTARGET_library`.